lib/src/widgets/cards/login_card.dart

```diff
@@ -284,11 +284,11 @@ class _LoginCardState extends State<_LoginCard> with TickerProviderStateMixin {
       width: width,
       loadingController: widget.loadingController,
       interval: _nameTextFieldLoadingAnimationInterval,
-      labelText: messages.userHint,
```

Comment:
This will break a custom userHint, and only restrict it to the ones given. It would be good if it can be overruled if the userHint is provided.